### ISSUE TITLE
No startup on Windows VM

### ISSUE THREAD

**Issue by Danielosama** [Madmanmayson/KanColleViewer#3]
Hey, first of all, thank you for the new KCV, I was losing my mind having to play on KC3Kai.

Now, it works without problems on my main computer, but when I tried to launch it on a VM, it will pop up for a few seconds on the Task Manager and then close without errors.

Any idea what it could be?

**Comment by Danielosama** [comment]
Windows 10 64bit
4Gb RAM
256Mb Video Memory
3 Processors

**Comment by Madmanmayson** [comment]
Since idk if it installs on a VM version of Windows 10, make sure .Net and VC++ is installed from here.
https://www.microsoft.com/en-US/download/details.aspx?id=30653
https://www.microsoft.com/en-us/download/details.aspx?id=48145

**Comment by Danielosama** [comment]
Installed both of them ( .NET was already installed), but the problem persists x(

**Comment by Madmanmayson** [comment]
hmmm I just did a fresh install of Windows 10 on a VM and had no issues. you sure the VM is 64 bit and not 32 bit? Idk if it works on 32 bit anymore after dependencies changed and switching to VS17

**Comment by Danielosama** [comment]
Well, here are my VM specs https://i.imgur.com/cktXS8o.png 
So unless this is lying to me, the VM is indeed Windows 10 64 bit.

But it is really weird, I never had problems with any version of KCV before, this is the first time.

**Comment by Madmanmayson** [comment]
is there any debug.log file?

**Comment by Danielosama** [comment]
Well, nevermind, I got it to work.
Uninstalled all VC++ and installed them again, works now. 
Guess I should have done that from the beginning.

However, I now have the same problem I had when I played on the VM via KC3Kai or directly through Chrome: the game uses 95%+ of the CPU and huge amounts of RAM, which results in an extremely laggy game x(
Playing with Edge though is a different story, 40% CPU usage at most, and low RAM usage.

Btw, on my main computer, KCV doesn´t use that much CPU and RAM at all, and runs just fine.
I admit I have no idea why it's like that, I'm not that knowledgeable about these things.

EDIT: And yes, there is a debug.log file now.

**Comment by Madmanmayson** [comment]
Well at least that part is fixed, unfortunately the viewer uses a chromium mini browser now so it will behave more like chrome in terms of memory usage. Its the only good alternative available to the old IE browser for C# and WPF. At that point its more looking into ways to speed up chrome rather then the program itself. (I personally use firefox for main internet use so idk much about why chrome is so taxing even though its the reason I switched). Going to close it for now though as the main issue was resolved.
